docs/how-to-build.md

```diff
@@ -21,31 +22,241 @@ The project itself includes the source code of several 3rd-party libraries, whic
   - libunwind
   - ncurses
   - readline
-  - bzip2
-  - doubleconversion
-  - fatal
-  - fbthrift
-  - folly
-  - gflags
-  - glog
-  - googletest
-  - jemalloc
-  - libevent
-  - mstch
-  - proxygen
-  - rocksdb
-  - snappy
-  - wangle
-  - zlib
-  - zstd
-
-### How to get 3rd-party Libraries
+
+#### Build and install 3rd-party Libraries
 [Install guide](https://github.com/vesoft-inc/nebula-3rdparty/blob/master/README.md)
 
 
-### How to build
+#### Build nebula
   - 1. create build dir : `cd nebula && mkdir build && cd build`
   - 2. generate makefile : `cmake -DNEBULA_THIRDPARTY_ROOT=${3rd-party_install_root} ..`
   - 3. make : `make or make -j${threadnum}`
   - 4. install: `make install`
   - notes : the default installation dir is **/usr/local/**, if you want to change the dir, on step 2, your command can be `cmake -DCMAKE_INSTALL_PREFIX=$your_nebula_install_dir ..`
+
+
+### How to build in Centos7.5, Centos6.5, Ubuntu18.04, Ubuntu16.04
+
+**code url**
```

Comment:
to be honest, I'm confused with the code url and dependence package url -- They are not the same catalog hierarchy 


![image](https://user-images.githubusercontent.com/50101159/63070441-ca5d2600-bf4d-11e9-8776-042dcb2b2491.png)


Is there any better way to install by going through directly. Try not to looking upside down backward.

It is better to give an direct command to install the depenency

for example

```
sudo apt install xx1,xx2,xx3.
```